deployments/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface IUnipoolContractConfig {
+    uniTokenAddress: string;
```

Comment:
We can add a `ERC20TokenConfig` that can be checked during deployment:

- `address` (obvious)
- `name`
- `symbol`
- `decimals`